src/jira/client/index.ts

```diff
@@ -182,7 +182,7 @@ async function getJiraClient(
 						});
 					}
 
-					await batchedBulkUpdate(
+					return await batchedBulkUpdate(
```

Comment:
Am I correctly understand you added return value so the `update` method will return a response object?

If we are adding return value here, we should add it everywhere in this file for consistency.

- [ ] make return values consistent for Jira API calls

Reply:
Where is it missing from? Just had a look at the file and it looks like every method either has a `return` statement or is being returned by `=>`

Reply:
Yes you are right, I didn't notice that there are no brackets